.github/workflows/tpu-tests.yml

```
@@ -3,7 +3,7 @@ name: Test PyTorch - TPU
 on:
   push:
     branches: [master, "release/*"]
-  pull_request_target:
+  pull_request:  # fixme: pull_request_target
```

Should this be reverted before merging this PR?
```suggestion
  pull_request:  # fixme: pull_request_target
```